### ISSUE TITLE
Crash when ergoemacs mode enabled twice in init.el and custom-set-variables

### ISSUE THREAD

**Issue by Valdaer** [ergoemacs/ergoemacs-mode#398]
Bug scenario:
-Fresh installed emacs from Debian repo with ergoemacs-mode from ELPA.
-enabled ergoemacs in init.el like guided here http://ergoemacs.github.io/
```
(setq ergoemacs-theme nil) ;; Uses Standard Ergoemacs keyboard theme
(setq ergoemacs-keyboard-layout "us") ;; Assumes QWERTY keyboard layout
(ergoemacs-mode 1)
```
if I remember correctly `(ergoemacs-mode 1)` would suffice

After first start/run(either by starting emacs with enabled ergoemacs-mode in init.el or by running ergoemacs-mode by command) everything is ok, but at the second start of emacs with above ergoemacs configuration in init.el it will crash because ergoemacs is configured by default to save itsefl configuration in custom-set-variables by the time of emacs shutting down, and after the first shutdown (ergoemacs-mode 1) is set in two places: in init.el by the user and in custom-set-variables.

To inhibit that default behavior of ergoemacs one must set:
```
(setq ergoemacs-theme-options (quote ((save-options-on-exit off))))
```
as I mentioned here: https://github.com/ergoemacs/ergoemacs-mode/issues/389

I think it should be fixed somehow or at least clearly mentioned in the wiki with that workaround because it is probably discouraging to many users who would like to use/try ergoemacs-mode. It was driving me nuts for a couple of days before I found out solution, because I didn't want to keep all my ergoemacs configuration in custom-set-variables.

**Comment by mattfidler** [comment]
I agree.  I haven't got to it yet.  I was planning on having menu-driven menu changes to start the customize engine, otherwise it would allow the user to set their own options.

**Comment by mattfidler** [comment]
May be addressed by 5827cfc15e2b34e47b2a95e1d3d87ff9b02a5df8

**Comment by mattfidler** [comment]
I haven't seen this for awhile.
